consul/kv_consul.go

```diff
@@ -930,7 +919,7 @@ func (kv *consulKV) watchTreeStart(
 	checkIndex := func(prevIndex *uint64, pair *api.KVPair, newIndex uint64,
 		msg string, lastIndex, waitIndex uint64) {
 		if *prevIndex != 0 && newIndex <= *prevIndex {
-			kv.FatalCb(msg+" with index invoked twice: %v, prevIndex: %d"+
+			logrus.Infof(msg+" with index invoked twice: %v, prevIndex: %d"+
```

Comment:
any reason why changing it from a panic callback to log?

Reply:
invalid connection was triggering fatal cb, which is now being mitigated by connection retry.